tas-easy-install.sh

```diff
@@ -50,6 +50,30 @@ install_sso_keycloak() {
         exit 1
     fi
 }
+
+# Generate the script to initialize the environment variables for the service endpoints
+generate_env_script() {
+    # Write the script to a file
+cat <<EOL > tas-env-variables.sh
+#!/bin/bash
+
+export BASE_HOSTNAME=apps.$(oc get dns cluster -o jsonpath='{ .spec.baseDomain }')
+echo "base hostname = \$BASE_HOSTNAME"
+
+export KEYCLOAK_REALM=sigstore
+export FULCIO_URL=https://fulcio.\$BASE_HOSTNAME
+export KEYCLOAK_URL=https://keycloak-keycloak-system.\$BASE_HOSTNAME
+export REKOR_URL=https://rekor.\$BASE_HOSTNAME
+export TUF_URL=https://tuf.\$BASE_HOSTNAME
+export KEYCLOAK_OIDC_ISSUER=\$KEYCLOAK_URL/auth/realms/\$KEYCLOAK_REALM
+EOL
+
+    # Make the generated script executable
+    chmod +x tas-env-variables.sh
+    echo "Environment variables script 'tas-env-variables.sh' has been created and made executable. NOTE: this must be executed by running `. tas-env-variables.sh` to update the env vars in the correct shell."
```

Comment:
ok so now I see you _did_ add that NOTE - but I missed it still - let's put it on its own line :) 

```
echo A script to set a local signing environment has been created in the current directory.
echo The script must run with \`dot space dot\` to set the environment of the current shell.
echo Run \'. ./tas-env-variables.sh\' from the terminal.
```